add tables and figures

vorugantia opened the pull request:
I added some figures and tables in this PR. They aren't fully fleshed out but they are a good start.

@cdeil - Can you check the ordering of survey images in the figure and also in the table? Is it okay or should we order them specifically (by name, low to high frequency band, etc.)? Also, I didn't completely fill in the table for survey images. Should we add/remove columns? Instead of frequency band, we could say "radio" or "x-ray" etc. Do you prefer that?

vorugantia committed:
add tables and figures

cdeil commented on proceeding/tables/catalog_table.tex:
```diff
@@ -0,0 +1,21 @@
+
+\caption{Catalog information.}
+\label{tab:a}
+\tabcolsep7pt\begin{tabular}{ || c|ccc ||}
```
I would drop the vertical lines and use alignment `lrll`, i.e. left for text and right for numbers.

cdeil commented on proceeding/tables/catalog_table.tex:
```diff
@@ -0,0 +1,21 @@
+% & \tch{1}{c}{b}{Single\\ outlet}  & \tch{1}{c}{b}{Small\\ multiple\tabnoteref{t1n1}}  & \tch{1}{c}{b}{Large\\ multiple}  & \tch{1}{c}{b}{Total}   \\
+
+
+\textbf{Catalog name} & \textbf{Number of sources} & \textbf{Update} & \textbf{Description} \\ \hline
```
Shorter titles to gain some space for description?
- Catalog name -> Catalog
- Number of sources -> Sources
  You can still mention that "number of sources" is given in the caption, so that it's clear.

cdeil commented on proceeding/tables/survey_table.tex:
```diff
@@ -0,0 +1,23 @@
+
+\begin{table*}[h]
+
+\caption{Survey information.}
```
I would suggest "Survey" -> "Image" here and for the column "Survey name" -> "Image" below.
Or use "Survey image".

Usually the product of a survey (like Planck) is a set of images and catalogs, so just using "survey" as caption for the images isn't the best choice IMO.

cdeil commented on proceeding/tables/survey_table.tex:
```diff
@@ -0,0 +1,23 @@
+\tabcolsep7pt\begin{tabular}{ || c|cc ||}
+\hline
+
+\textbf{Survey name} & \textbf{Telescope/Observatory} & \textbf{Frequency band}\\ \hline
```
I think this set of columns might work better:
- Image (what's Survey name now)
- Resolution (roughly indicate angular resolution in deg or arcmin, I can help with that)
- Band (or "Wavelength" or "Energy")
- Color? (is it color or grayscale)
- Description -- put the Telescope/Observatory info here.

What do you think?
Have resolution, band and color as separate columns?
Or have them part of the description sentence?

vorugantia replied:
I like them as separate columns. I made the changes.

cdeil commented on proceeding/tables/survey_table.tex:
```diff
@@ -0,0 +1,23 @@
+\textbf{IRIS Band 4-100um} & IRIS & \\
+\textbf{Planck R2 LFI Color 30-44-70 GHz} & Planck & 30-44-70 GHz\\
+\textbf{Planck R1 HFI Color 353-545-857 GHz} & Planck & 353-545-857 GHz\\
+\textbf{Planck R2 HFI Color 353-545-857 GHz} & Planck & 353-545-857 GHz\\
```
Maybe combine the Planck ones and mention that there's several in the description?
This is anyways just a first selection, I think this will change a lot in the coming months.

cdeil replied:
Yes, I think maybe just putting "radio", "x-ray", "gamma-ray" might be better for a quick overview than giving quantitative wavelength / freq / energy info (put those in the description if there is space).

cdeil commented on proceeding/text/data.tex:
```diff
@@ -31,4 +29,12 @@ \section{Data}
+
+\begin{figure}[h]
+  \centerline{\includegraphics[width=\textwidth]{figures/four_images}}
+  \caption{Survey images (left to right): Fermi color, Akari 90um, Planck LFI, Planck HFI.}
```
This I find very interesting. Nice!
Maybe even show a second row of spheres?
Or have a second row with zoom-ins for the Galactic center region (or some other interesting region), like 20 x 10 deg or 10 x 5 deg, maybe even with gamma-ray sources shown?
That would already to in the direction of "science application", we would write a few sentences comparing the visible structures.

cdeil replied:
Maybe also put the label of which sphere is which on the image? Then it can easily be copied onto posters and slides and is quicker to grok.

cdeil commented on proceeding/text/features.tex:
```diff
@@ -37,11 +35,19 @@ \section{Features}
+
+\begin{figure}[h]
+  \centerline{\includegraphics[width=\textwidth]{figures/catview_wide}}
+  \caption{Catalog View.}
```
The catalog view image is a bit small. Remove white space on left and right so that the content is better visible?

cdeil commented:
Very nice, thanks!

I left some inline comments.

You could also already in this PR add short captions for the figures and tables (it should be one or two rows, max three, more detailed info is then in the main text).

And you could already add `\ref` from the main text to the Figures / tables, so that the relative order of them and structure of the paper becomes clear.

(or leave those extra things to future PRs if you prefer.)

vorugantia committed:
fix tables, add vela region image

vorugantia committed:
update

vorugantia commented:
@cdeil:

Made Catalog View screenshot bigger.

Fixed table text alignment and removed the vertical line. Fixed naming issues.

See the new image table. For image names, I just copied the names from CDS' HiPS database. However, they seem to be very long and redundant of other columns (e.g. Planck images contain "Color" and frequency band in their names, but that info is already in other columns). Also, since on our website you can change the color map for most images, do we need a Color column? One more issue is the Resolution column, where CDS' HiPS database said the image resolution for all images was the same (except GLIMPSE360 but I looked that one up online). That can't be correct though, right?

Added captions to figures of survey images. Also check out the Vela region figure and let me know if I should change anything.

I'll make one final round of changes and then merge this PR.

cdeil commented on proceeding/tables/catalog_table.tex:
```diff
@@ -0,0 +1,21 @@
+
+\caption{Catalog information.}
+\label{tab:a}
+\tabcolsep7pt\begin{tabular}{ || lrll ||}
```
I think you should also remove the outer vertical lines.
I've never seen this in publications, and it looks weird / unnecesary to me.

cdeil commented on proceeding/tables/catalog_table.tex:
```diff
@@ -0,0 +1,21 @@
+% & \tch{1}{c}{b}{Single\\ outlet}  & \tch{1}{c}{b}{Small\\ multiple\tabnoteref{t1n1}}  & \tch{1}{c}{b}{Large\\ multiple}  & \tch{1}{c}{b}{Total}   \\
+
+
+\textbf{Catalog} & \textbf{Sources} & \textbf{Update} & \textbf{Description} \\ \hline
```
Remove all the manual `\textbf`?
(I think it's uncommon, plus does it really help with readability?)

cdeil commented on proceeding/tables/catalog_table.tex:
```diff
@@ -0,0 +1,21 @@
+\textbf{3FGL} & 3034 & fixed & Third Fermi-LAT point source catalog\\
+\textbf{2FHL} & 360 & fixed & Second Fermi-LAT Catalog of High-Energy Sources\\
+\textbf{SNRcat} & 378 & continuous & A census of high-energy observations of Galactic supernova remnants\\
+\textbf{TeV} & 155 & continuous & The TeGeV Catalogue @ ASDC\\
```
Maybe put in this order?
TeV, 2FHL, 3FGL, SNRcat

Or at least group the gamma-ray catalogs together, and put the MWL catalogs (only SNRCat at the mometn) below.

cdeil commented on proceeding/tables/image_table.tex:
```diff
@@ -0,0 +1,22 @@
+\textbf{Haslam 408} & 51.53 & radio & grayscale & 408 MHz\\
+\textbf{IRIS Band 4-100um} & 51.53 & infrared & grayscale & IRIS \\
+\textbf{Planck R2 LFI Color 30-44-70 GHz} & 51.53 & microwave & color & Planck 30-44-70 GHz\\
+\textbf{Planck R1 + R2 HFI Color 353-545-857 GHz} & 51.53 & microwave & color & Planck 353-545-857 GHz\\
```
I agree, these are way too long.
Invent your own short label (like you did for the catalogs), and put all other info (if really needed) in the "description".

cdeil commented on proceeding/text/data.tex:
```diff
@@ -31,4 +29,19 @@ \section{Data}
+
+\begin{figure}[h]
+  \centerline{\includegraphics[width=\textwidth]{figures/vela_region_captions}}
+  \caption{The Vela Region in various survey images and color maps.}
```
Mention what the image center and (approximate) field of view is?

cdeil commented:
I left some inline comments.

> One more issue is the Resolution column, where CDS' HiPS database said the image resolution for all images was the same (except GLIMPSE360 but I looked that one up online). That can't be correct though, right?

Yes, that info doesn't reflect the angular resolution of the survey.
At http://alasky.u-strasbg.fr/Haslam408/ it says:

> Best pixel angular resolution: 51.53"
> but that is just the best sampling resolution they used when making the HIPS map.
> It's a different number from the point spread function angular resolution of the survey.

For the Haslam survey the paper says:
http://adsabs.harvard.edu/cgi-bin/bib_query?1982A&AS...47....1H

> The angular resolution is 0°.85.

Probably we have to go to the paper or webpage in all cases to get this number.
I think it's worth it, it'll make our description more interesting to gamma-ray astronomers, if they see other MWL surveys they are interested in, so it's OK if we spend time on making that table good (and selecting some other surveys).

Please split that out into a different reminder issue, I can look up the resolutions later.

cdeil commented:
Concerning the choice of "sky region of interest" where you chose Vela.

If you have time, could you make similar MWL image collages for other potentially interesting regions?
- Galactic center
- Cygnus
- Large magellanic cloud
  For now, these could go on the data wiki page (and later link to the interactive map once we have routing for the map). And then we decide which is the most interesting and pick that one for the paper.
  If you don't have time for this in the coming ~ week, please copy this into a separate reminder issue for us to do later.

Otherwise, 👍  to merge.

vorugantia commented:
@cdeil:

I changed the "description" column in the image table to just "band", because all I had for the description column was frequency band. We should probably fill out the rest of that column for the other images, though.

I changed the ordering of catalogs to what you suggested. I also changed the ordering of images to alphabetical.

Reminder issue created to look up angular resolution.

Added 3 more "sky region of interest" photos to the document for now. You can just pick out a favorite and remove the others from the document.

Merging this PR...

vorugantia committed:
update
